src/wtforms/form.py

```diff
@@ -223,7 +237,14 @@ class Form(BaseForm, metaclass=FormMeta):
     Meta = DefaultMeta
 
     def __init__(
-        self, formdata=None, obj=None, prefix="", data=None, meta=None, **kwargs
+        self,
+        formdata=None,
+        obj=None,
+        prefix="",
+        data=None,
+        meta=None,
+        extra_filters=None,
```

Comment:
Should this be a parameter? `extra_validators` isn't.

Reply:
You are right, I am not sure why I changed that...